src/odemis/gui/comp/overlay/base.py

```diff
@@ -239,10 +239,10 @@ def draw(self, ctx, canvas_padding=None, view_width=None, view_height=None):
 
         # Draw background
         if self.background:
-            margin_x = 4  # margins for better representation of background
-            margin_y = 6
-            ctx.set_source_rgba(0.0, 0.0, 0.0, 0.7 * self.opacity)
-            rect = (x - margin_x, y + margin_y, self.text_size[0] + margin_x, -self.text_size[1] - margin_y)
+            margin_x = 6  # margins for better representation of background
+            margin_y = 10
+            ctx.set_source_rgba(*(self.background + (self.opacity,)))
```

Comment:
That makes sense. However, notice that overlay.view.PixelValueOverlay currently sets the background to 4 floats. So this change breaks it. I'd recommend to adjust PixelValueOverlay (line 240) to use `hex_to_frgb()`. 
Alternatively, you could make it even more clever and detect that there is a 4th value, and so use it to multiply with `self.opacity`. This way, the caller can still choose to have a semi-transparent background (as it was so far) while the letters are fully opaque.